src/data_storage/import/container_import.ts

```diff
@@ -428,22 +644,48 @@ export default class ContainerImport {
                   max
                 },
                 options: propertyOptions,
-                defaultValue: ""
```

Comment:
is the compiler forcing you to include this? because as it stands this will record as a blank string in the database. This should stay undefined.

Reply:
Compiles fine without that property, I'll go ahead and remove it.